templates/experiments/experiment_form.html

```diff
@@ -16,35 +19,62 @@
   </div>
 {% endblock %}
 {% block form %}
-  {{ form.non_field_errors }}
-  {% flag "assistants" %}
-    {% render_form_fields form "name" "description" %}
-    <h3 class="font-semibold text-xl mt-4">Language Model Configuration</h3>
-    <div class="divider divider-neutral"></div>
-    <div class="pl-4">
-      <div role="tablist" class="tabs tabs-bordered mt-4">
-        <input type="radio" name="bot_type_tabs" role="tab" class="tab" aria-label="Use LLM Directly" {% if not object.assistant_id %}checked{% endif %}/>
-        <div role="tabpanel" class="tab-content pt-4">
-          {% render_form_fields form "llm_provider" "llm" "temperature" "prompt_text" "tools_enabled" "source_material" %}
-        </div>
+  <div>
+    {% render_form_fields form "name" "description" "consent_form" %}
+    <div class="divider"></div>
+    {% render_form_fields form "type" %}
+    {% if not form.fields.assistant.choices %}
+      <div x-show="type === 'assistant'">
+        <p class="text-error">{% blocktranslate %}
+          No assistants have been created. Please create an assistant before creating an 'assistant' experiment.
```

Comment:
Nit: How about this (in this or a future PR): "There are no assistants available to use. <a>Click here</a> to create one."